### PR TITLE
Bumped Pyo3 to 0.20

### PR DESCRIPTION
**Pull request by remusao**

### CODE REVIEW

File: src/de.rs [hunk 1]
```diff
@@ -258,7 +258,7 @@ impl<'a, 'de> de::Deserializer<'de> for &'a mut Depythonizer<'de> {
                 .get_item(0)?
                 .downcast()
                 .map_err(|_| PythonizeError::dict_key_not_string())?;
-            let value = d.get_item(variant).unwrap();
+            let value = d.get_item(variant).unwrap().unwrap();
```
Comment (remusao) [inline src/de.rs:261]: I am not 100% sure this is the correct way to handle such case.

Comment (davidhewitt) [reply src/de.rs:261]: I think this can work, and if so, is better:

```suggestion
            let value = d.get_item(variant)?.unwrap();
```

Comment (remusao) [reply src/de.rs]: It seems to work :+1: Thanks for the suggestion.

**Comment by remusao** [comment]
I will have a look into the failing tests.

**Comment by remusao** [comment]
I have updated the PR with more changes so that tests are green (at least locally).

**Review by davidhewitt — APPROVED** [review]
Looks great to me, thanks and I'll try to get this released ASAP

**Comment by Vonfry** [comment]
What is the plan for the next release?

**Comment by davidhewitt** [comment]
This has been released in pythonize 0.20

**Pull request merged**